<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassword</class>
 <widget class="QWidget" name="changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>479</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>479</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/private_icon/private_logo_2.png</normaloff>:/private_icon/private_logo_2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;Hack&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit_newPasswordConfirm_input">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit_oldPassword_input">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QLabel" name="label_changeStatusBar">
     <property name="minimumSize">
      <size>
       <width>451</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>NEW PASSWORD:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit_newPassword_input">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_runChange">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>211</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>change password</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButton_cancelChangeProccess">
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>211</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>canlcel proccess</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>461</width>
       <height>36</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Change Login Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>OLD PASSWORD:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>CONFIRM NEW PASSWORD:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="text">
      <string>USERNAME:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit_username_input">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>461</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Hack&quot;;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main_icon_files.qrc"/>
 </resources>
 <connections/>
</ui>
